ios/GatewayManager.m

```diff
@@ -64,6 +65,45 @@ - (void)configureStripeGateway:(NSDictionary *_Nonnull)gatewayParameters
 #endif
 }
 
+
+- (void)savePaymentMethod:(NSString *)clientSecret cardParams:(NSDictionary *)cardParams completion:(void (^)(NSString * _Nullable, NSError * _Nullable))completion
+{
+#if __has_include(<Stripe/Stripe.h>)
+    // Collect card details
+    STPPaymentMethodCardParams *card = [[STPPaymentMethodCardParams alloc] init];
+    card.number = [RCTConvert NSString:cardParams[@"number"]];
+    card.expYear = [RCTConvert NSNumber:cardParams[@"expYear"]];
+    card.expMonth = [RCTConvert NSNumber:cardParams[@"expMonth"]];
+    card.cvc = [RCTConvert NSString:cardParams[@"cvc"]];
+    STPPaymentMethodParams *paymentMethodParams = [STPPaymentMethodParams paramsWithCard:card billingDetails:nil metadata:nil];
```

Comment:
Maybe here, `paramsWithCard` could potentially be `paramsWithToken`

Reply:
If it can I don't know https://stripe.dev/stripe-ios/docs/Classes/STPPaymentMethodParams.html